service/test/POSIXSignalTest.cpp

```diff
@@ -201,10 +243,13 @@ TEST_F(POSIXSignalTest, sig_queue_ESRCH)
  */
 TEST_F(POSIXSignalTest, sig_queue_EPERM)
 {
-    if (geteuid() == 0) {
+    if (geteuid() == 0) {  // the root user
         std::cerr << "Warning: <geopm> Skipping POSIXSignalTest.sig_queue_EPERM cannot be run by user \"root\"\n";
     }
-    else {
+    else if (has_elevated_permissions()) {  // the non root user with elevated permissions
+        m_posix_sig->sig_queue(1, SIGCONT, 2);  // TODO: how to do EXPECT_NO_THROW?
```

Comment:
How to do `EXPECT_NO_THROW`? Like for any other non root user, we used a `GEOPM_EXPECT_THROW_MESSAGE` to say that we are expecting `sig_queue()` to necessarily fail. So if it doesn't throw an exception, the test would fail.

For the non root user with elevated permissions, can we have a similar assertion that excepts the call `sig_queue()` to necessarily **not** fail, or necessarily **not** throw an exception. So if it throws an exception, the test would fail, and if it doesn't throw, the test would pass. How can we do that?

Reply:
If the test throws an unhandled exception, gtest considers it a failure, that is why there isn't an "EXPECT_NO_THROW" macro.  Everything is expected not to throw :)

Reply:
> If the test throws an unhandled exception, gtest considers it a failure, that is why there isn't an "EXPECT_NO_THROW" macro. Everything is expected not to throw :)

So nothing else needs to be done here?